docs/how_to_run.md

```diff
@@ -0,0 +1,9 @@
+## Scenarios
+
+### [Perception Pipeline Benchmark](scenarios/perception_pipeline_benchmark.md)
+
+This benchmark measures the elapsed time by which the determined path is sent for the robot to follow. This benchmark calculates `elapsed time`, `success_number` and `failure_number`. `elapsed_time` is used for measuring how much time the pipeline takes. `success_number` is used for presenting successfully plannings and `failure_number` is used for presenting failed plannings.
```

Comment:
How to you differentiate between time required by the middleware communication layer  vs the planner/ execution algorithms?

Reply:
The elapsed time means real time in benchmark results created by google benchmark tool. It seems there is no other way to measure the time plan-execution jobs took and the time data communication jobs took. Instead, i follow the strategy that 

* to measure elapsed time (wall time, real time in other meaning)  
  * send pose msg for the robot to plan
  * wait for servers response (data communication jobs + planning jobs) 
  * send plan msg for robot to execute trajectory (data communication jobs + execution jobs) 
* to do this measurements per each middleware
* compare the results

If we make the total time of planning jobs and execution jobs same for each middleware, planning jobs and execution jobs don't affect comparison of middleware. To send much goal pose (i'm speaking for perception pipeline) decreases randomness of ompl planner etc. The real aim isn't to mesaure the time, but to compare the time per each middleware.